reflection/serverreflection.go

```diff
@@ -144,7 +266,7 @@ func (s *serverReflectionServer) fileDescContainingExtension(st reflect.Type, ex
 		return nil, fmt.Errorf("failed to find registered extension for extension number %v", ext)
 	}
 
-	return s.decodeFileDesc(proto.FileDescriptor(extDesc.Filename))
+	return decodeFileDesc(proto.FileDescriptor(extDesc.Filename))
 }
```

Comment:
Could we change this method into a function as well, since it doesn't use any fields from the struct?

Reply:
Yes. Done.